cat-home/src/test/java/com/dianping/cat/report/task/TaskHelperTest.java

```diff
@@ -50,7 +62,14 @@ public void testYesterdayZero() {
 		cal.set(Calendar.SECOND, 0);
 		cal.set(Calendar.MILLISECOND, 0);
 		Date yesterdayZero = TaskHelper.yesterdayZero(cal.getTime());
-		Assert.assertEquals(cal.getTimeInMillis(), yesterdayZero.getTime() + 3600L * 1000 * 24);
+		Assert.assertEquals(cal.getTimeInMillis(),
```

Comment:
This is an existing test which we are not adding to and it appears they are happy to have long lines, so I don't think we should be wrapping this line.